cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -209,12 +207,6 @@
         "Tenancy": {
           "default": "tenancy"
         },
-        "EphemeralKMSKeyId": {
-          "default": "ephemeral_kms_key_id"
-        },
-        "ClusterReadyScript": {
-          "default": "cluster_ready"
```

Comment:
After this patch remember to cleanup the cookbook according to this:
- [files/default/compute_ready](https://github.com/aws/aws-parallelcluster-cookbook/blob/v1.6.0/files/default/compute_ready)
- [recipes/_prep_env.rb](https://github.com/aws/aws-parallelcluster-cookbook/blob/v1.6.0/recipes/_prep_env.rb#L45)

Reply:
Very good point! Thanks